index.html

```diff
@@ -0,0 +1,157 @@
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,100;1,300;1,400&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+    <title>Dbn Hiphop Fest</title>
+    
+</head>
+<body>
+    <header id="header section">
+          <a class="Logo logo1" href="#">Techmoves</a>
+          <a class="Logo10" href="#">Techmoves</a>
+        <nav>
+     <div class="mobile-menu">       
+            <img class="humburger" src="images/hum1.png" alt="humburger">
+            <ul>
+              <li><a class="col-nav1" href="./index.html">Home</a>
+            </li>
+              <li><a class="col-nav1" href="./about.html">About</a>
+            </li>
+              <li><a href="#main-program-sec">Feature</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#">Updates</a></li>
+              <li><a class="col-nav" href="#">CC campaign</a></li>
+            </ul>
+         </div>
+        </nav>
+    </header>
+     <div class="option">
+        <button id="closebtn">&times;</button>
+        <ul>
+           <li><a class="col-nav1" href="./index.html">Home</a>
+            </li>
+            <li><a class="col-nav1" href="./about.html">About</a>
+            </li>
+            <li><a href="#main-program-sec">Feature</a></li>
+        </ul>
+    </div> 
+    <section id="top-sections">
+        <div class="notes">
+                <p class="great-text" >"Techmoves Records Brings"</p>
+   <h1>Durban hip hop fest 2023</h1>
+<div class="center-main">
+
+<article>It time to elevete the culture of Durban Hip hop that gonna takes place on 15 of june 2023 from 11 am, til 12am at KingSpark Stadium.go get ticket at coputerticket and ticket won't be sold at the door</article>  
+ </div>
+<div class="center-main1">
+
+ <h2 class="date">
+    2023.06.15(SAT) ~ Till Late
+   
+</h2>
+<span class="loaction">@KingSpark Stadium</span>
+ </div>     
+ </div>
+   
+</section>
+<section id="main-program-sec">
+  
+    <h3 class="inter-act1">Main Feature</h3>
+    <hr class="horizontal-main-line">
+
+ <div class="main-feature">
+    <div class="guest">
+<div class="activity">
+    <img class="pro"src="images/boobat.png" alt="icon-guest">
+    <a href="#">Boombat Rap</a>
+    <p class="event-details">
+        Listen to the speakers from various countries about the messages of sharing and opening.
+    </p>
+</div>
+
+<div class="activity">
+    <img class="pro"src="images/dance.png" alt="icon-guest">
+    <a href="#">Dancing Stage</a>
+    <p class="event-details">
+        Listen to the speakers from various countries about the messages of sharing and opening.
+    </p>
+</div>
+<div class="activity">
+    <img class="pro"src="images/merch.png" alt="icon-guest">      
+    <a href="#">Merch</a>
+    <p class="event-details">
+        Listen to the speakers from various countries about the messages of sharing and opening.
+    </p>
+</div>
```

Comment:
Please let's consider using an `article` tag instead of the generic `div` tag for this section for semantic reasons. 💪🛠

kindly use a header `hx` tag for the title of the individual articles, e.g. `<a href="#">Main Stage</a>` that way it will be semantically correct. 👍

Reply:
resolved